Tech Journey/My-#100DayOfCode-Journey-So-Far.md

```diff
@@ -0,0 +1,56 @@
+## A bit about me, Who am I?
+Hey, my name is **Ameen**, I am 15 years old and currently in high school. I started learning to code when I was in second grade in school but never took it seriously until I was 11... That time I started learning **Unity** ( A Game Engine ) and **C#** but due to my PC's specs I couldn't continue on that path so instead of giving up  I started learning **Python**, learned **PyGame** and a few more libs and built something basic with each of em.
+But I didn't enjoy it as much and started to lose my interest in python. So the hunt to find something fun had begun again. After exploring for a while, I found "**Web Development**", when I started learning web development it just clicked with me didn't take me much time to master HTML but then school came in the way . 2 years later I again continued the web dev path with the #100DaysOfCode challenge and am having fun ever since.
+I currently know HTML, CSS, js, and learning svelte and built around 12-18 projects alongside with it. That's enough about me let's get into the challenge.
+<hr>
+## What is **#100DaysOfCode** ??
+**#100DaysOfCode** is a challenge where you have to code ** 1-hour** at minimum for **100** days straight. The only reason you can take a break is if you are sick, need to travel, or just want to relax or take a break, just try not to take a break for more than 2 days at once.
+That's pretty much it plans to master something and code for 100 days straight and even tweet your progress on twitter :)
```

Comment:
Again, too many ands. As a general rule, avoid more than one and in any sentence. If you're saying two different things, say it in two different sentence.